plugins/flashrom-i2c/flashrom-i2c.quirk

```diff
@@ -0,0 +1,4 @@
+[HwId=1AF80175]
+Programmer=lspcon_i2c_spi
+Device=PS175
+Protocol=com.parade.ps175
```

Comment:
Is the protocol specific to the PS175? It looks like a generic flashrom blob to me.